Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -36,7 +36,10 @@
 
 NSString *const BUYApplePayPaymentProviderId = @"BUYApplePayPaymentProviderId";
 
-@interface BUYApplePayPaymentProvider () <PKPaymentAuthorizationViewControllerDelegate>
+typedef void (^AddressUpdateCompletion)(PKPaymentAuthorizationStatus, NSArray<PKShippingMethod *> * _Nonnull, NSArray<PKPaymentSummaryItem *> * _Nonnull);
```

Comment:
We should probably stay consistent with `typedef`s across the SDK. This should probably be prefixed with `BUY`.